gtsam/hybrid/tests/testHybridGaussianFactor.cpp

```diff
@@ -54,32 +54,49 @@ TEST(HybridGaussianFactor, Constructor) {
   CHECK(it == factor.end());
 }
 
+/* ************************************************************************* */
+namespace testA {
```

Comment:
I would prefer this namespace being called `test_constructor` or `test_constructor_fixture` so it is obvious what it is being used for.

Reply:
Done